target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -11,6 +11,13 @@ board_config_update
 board=$(ipq806x_board_name)
 
 case "$board" in
+ad7200)
+	ucidef_set_led_usbdev "usb1" "USB 1" "${board}:blue:usb_1" "2-1"
+	ucidef_set_led_usbdev "usb2" "USB 2" "${board}:blue:usb_3" "4-1"
```

Comment:
Yep, you are partly correct: usb1-port1 (USB2.0) and usb2-port1 (USB3.0) are physical port 1, usb3-port1 (USB2.0) and usb4-port1 (USB3.0) are physical port 2